.github/workflows/native-full_marian-ubuntu.yml

```diff
@@ -87,34 +61,89 @@ jobs:
         CC=/usr/bin/gcc-${{ matrix.gcc }} CXX=/usr/bin/g++-${{ matrix.gcc }} CUDAHOSTCXX=/usr/bin/g++-${{ matrix.gcc }} \
         cmake .. \
           -DBoost_ARCHITECTURE=-x64 \
-          -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_BUILD_TYPE=Slim \
           -DCOMPILE_CPU=${{ matrix.cpu }} \
           -DCOMPILE_CUDA=${{ matrix.gpu }} \
-          -DCOMPILE_EXAMPLES=on \
-          -DCOMPILE_SERVER=on \
+          -DCOMPILE_EXAMPLES=off \
```

Comment:
`COMPILE_EXAMPLES` is set to `on` [here](https://github.com/browsermt/bergamot-translator-tests#jenkins). Am I missing something?

Reply:
Don't need the example binaries over here. I copied over the old marian workflow files while I was less knowledgeable regarding the ecosystem to do something which was minimum invasive. 

Since the tests at browsermt/marian-dev covers that these build anyway and I'm not using the binaries for any tests here, I see no point in generating these (some time is saved as well in build I hope). Will set them back if and when required in future (ie if I use them in tests).

The overall build time in this path has been brought down from 30m to 19m with these optimizations (as this was making progress in this PR slow).

Reply:
Makes sense 👍 

In this case, we should update the [README](https://github.com/browsermt/bergamot-translator-tests#readme) in bergamot-translator-tests repository because it is misleading 😄